packages/store/src/utils/selector/selector-processor.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  ensureSelectorMetadata,
+  getSelectorMetadata,
+  getStoreMetadata,
+  globalSelectorOptions,
+  SelectFromState,
+  SelectorMetaDataModel,
+  SelectorType,
+  SharedSelectorOptions,
+  StateClassInternal
+} from '../../internal/internals';
+import {
+  CreationMetadata,
+  RuntimeSelectorInfo,
+  selectorOptionsMetaAccessor
+} from './selector.tokens';
+
+export class SelectorProcessor {
```

Comment:
Please don't make a class. I can see your Java habits coming through here 😆 
These can just be exported functions.

Reply:
In the light of the main comment this file grouping doesn't make sense either.